docs/docs/configuration/object_detectors.md

```diff
@@ -309,14 +309,23 @@ RKNN support is provided using the `-rk` suffix for the docker image. Moreover,
 ### Configuration
 
 This `config.yml` shows all relevant options to configure the detector and explains them. All values shown are the default values (except for one). Lines that are required at least to use the detector are labeled as required, all other lines are optional.
+
 ```yaml
 detectors:                            # required
   rknn:                               # required
     type: rknn                        # required
+    # core mask for npu
+    core_mask: 0
+    # yolov8 model in rknn format to use; allowed calues: n, s, m, l, x
+    yolov8_rknn_model: n
+    # minimal confidence for detection
+    min_score: 0.5
+    # determines whether two overlapping boxes should be combined
+    nms_thresh: 0.45
 
 model:                                # required
   # path to .rknn model file
-  path: /models/yolov8n-320x320.rknn
+  path:
```

Comment:
there is still a default here right?

Reply:
Actually, the default is `None`. The image provides yolov8{n,s,m,l,x} models that can be chosen using the `yolov8_rknn_model`. 

This option allows the user to mount his own rknn model to the container and use it.

Reply:
I think providing two separate mechanisms for users to select a different model is going to be confusing. I'd expect that we can make this work with the current mechanism

I would suggest that you could do something similar to TRT where the user can choose from one of the rknn models using an environment variable and that model will be downloaded before frigate starts up

Reply:
Hm, but this way the user has to configure the yolov8 model using the docker-compose.yml. What if the user can pass something like "default-yolov8-nano", "default-yolov8-small", ... using the `model: path:` option if he wants to use one of the provided models and an actual path like "/config/model_cache/rknn/my-model.rknn" if he wants to use one model that he mounts to the container? This way everything can be done in the config using just the `model: path:` option.

Reply:
that could work as well

Reply:
Alright, I would actually prefer this way. Thank you for your feedback.